doc/arch/adr-015-convention-for-data-warehouse-unique-id.md

```diff
@@ -0,0 +1,44 @@
+# Use compound id instead of a random UUID
+
+26-Sep-2018
+
+## Context
+
+In the Data Warehouse [we have decided to track content through time and use a unique identifier](https://github.com/alphagov/content-performance-manager/pull/924) that guarantees consistency across time.
+
+Our initial approach was to use a random uuid:
+
+```ruby
+ SecureRandom.uuid #=> "1ca71cd6-08c4-4855-9381-2f41aeffe59c"
+```
+
+Following some feedback from @tijmenb, we have decided to use a compound id for each on of our items. The compound id will have the following format depending of the type of Content Item:
+
+### Single content items (no multi-path)
+
+We concat the `content_id` and the `locale` for Content Item that don't have multiple paths. 
+
+```ruby
+  warehouse_uid = "1ca71cd6-08c4-4855-9381-2f41aeffe59c:en"
```

Comment:
Should `warehouse_uid` be `warehouse_item_id`?

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/939/files#diff-fc9e31193c10d7d1347a6bfabcd2b441R12